examples/tensorflow/requirements.txt

```diff
@@ -1,4 +1,50 @@
-promote==1.10.0
-tensorflow==1.8.0
-numpy==1.14.3
-pillow==5.1.0
+git+https://git@github.com/alteryx/promote-python@python310
+absl-py
+astunparse
+cachetools
+clang
+contextlib2
+cycler
+flatbuffers
+gast
+google-auth
+google-auth-oauthlib
+google-pasta
+graphviz
+grpcio
+h5py
+keras
+Keras-Preprocessing
+Markdown
+numpy
+oauthlib
+opt-einsum
+pandas
+Pillow
+protobuf
+pyasn1
+pyasn1-modules
+pydot
+pydotplus
+pyparsing
+python-dateutil
+python-utils
+pytz
+requests
+requests-oauthlib
+rsa
+scipy
+schema
+six
+tensorboard
+tensorboard-data-server
+tensorboard-plugin-wit
+tensorflow
```

Comment:
do you need to specifiy a version of tensorflow? this model isn't working for me:
```
Traceback (most recent call last):
  File "cnn_mnist.py", line 13, in <module>
    tf.logging.set_verbosity(tf.logging.INFO)
AttributeError: module 'tensorflow' has no attribute 'logging'
```

Reply:
cnn_mnist.py only work with old version of tensoflow. I added a new file mnist.py for the upgraded version of tensorflow

Reply:
okay - tested the mnist.py. file and had to add `matplotlib` and graphviz to get it to run. - can you add that to the requirements.txt and readme? you have to install graphviz from here: https://graphviz.gitlab.io/download/

Reply:
The promote.sh will install graphviz. And matplotlib is install automatically with graphviz? I don't remember.  I will update readme and requirements

Reply:
also maybe we should just remove the cnn_mnist file? or at least note in the README that you should use the other file

Reply:
yeah, I just updated readme file to use mnist.py for training. leave cnn-mnist.py there and noticed it's for tensorflow < 2